docs/lake/design.md

```diff
@@ -121,13 +129,12 @@ Data is then loaded into a lake with the `load` command, .e.g.,
 zed lake load -p logs sample.ndjson
 ```
 where `sample.ndjson` contains logs in NDJSON format.  Any supported format
-(i.e., CSV, JSON, NDJSON, Parquet, ZNG, and ZST) as well multiple files can be used
-here, e.g.,
+(NDJSON, ZNG, ZSON, etc.) as well multiple files can be used here, e.g.,
 ```
-zed lake load -p logs sample1.csv sample2.ndjson sample3.zng
+zed lake load -p logs sample1.ndjson sample2.zng sample3.zson
```

Comment:
The example that was here previously wouldn't work because CSV is _not_ auto-detected currently, and it looks like when specifying `-i` we only accept one format at a time. I therefore replaced the prior "i.e.," (which read as if the list that followed was complete) with the closing ", etc.") (which, to me, reads more like an incomplete list, which I prefer). In addition to encouraging the reader to read on and learn about the `-i` needed for the other supported formats, I figure this hand-waves at the not-explicitly-mentioned supported Zeek TSV format. I respect the purity of not calling out Zeek explicitly, but in the short term a lot of the users are likely to be importing that, so I'm kinda playing both sides here. Thankfully most users who've come to this doc (such as the ones I'm currently engaging with on the #beta channel on Slack) already use `zq` and have come to expect Zeek TSV will "just work", so I'm not too concerned here.